source/text/Font.cpp

```diff
@@ -127,19 +142,29 @@ void Font::DrawAliased(const DisplayText &text, double x, double y, const Color
 
 
 
-void Font::Draw(const string &str, const Point &point, const Color &color) const
+void Font::Draw(const string &str, const Point &point, const Color &color, bool matchBack) const
 {
-	DrawAliased(str, round(point.X()), round(point.Y()), color);
+	DrawAliased(str, round(point.X()), round(point.Y()), color, matchBack);
 }
 
 
 
-void Font::DrawAliased(const string &str, double x, double y, const Color &color) const
+void Font::DrawAliased(const string &str, double x, double y, const Color &color, bool matchBack) const
```

Comment:
```suggestion
void Font::DrawAliased(const string &str, double x, double y, const Color &color, bool matchBack, int backBuffer) const
```

Make the function take a framebuffer and get rid of the FrameBuffer::GetTexture() method.